addon/templates/components/polaris-resource-list/item.hbs

```diff
@@ -1,140 +1,162 @@
+      <div class="Polaris-ResourceList-Item__Owned">
+        {{#if this.selectable}}
+          {{#let (element "div") as |ItemHandle|}}
+            <ItemHandle
+              data-test-id="larger-selection-area"
+              class="Polaris-ResourceList-Item__Handle"
+              onclick={{fn this.handleLargerSelectionArea}}
+            >
+              {{#let (element "div") as |CheckboxWrapper|}}
+                <CheckboxWrapper
+                  class="Polaris-ResourceList-Item__CheckboxWrapper"
+                  onclick={{fn this.stopPropagation}}
+                >
+                  <PolarisCheckbox
+                    @id={{this.checkboxId}}
+                    @label={{label}}
+                    @labelHidden={{true}}
+                    @checked={{this.isSelected}}
+                    @disabled={{this.loading}}
+                    @onChange={{fn this.handleSelection}}
+                  />
+                </CheckboxWrapper>
+              {{/let}}
+            </ItemHandle>
+          {{/let}}
+        {{/if}}
 
-  {{#if (or hasBlock children)}}
-    <div class="Polaris-ResourceList-Item__Content">
-      {{#if hasBlock}}
-        {{yield}}
-      {{else}}
-        {{render-content children}}
-      {{/if}}
-    </div>
-  {{/if}}
+        {{#if @media}}
+          <div class="Polaris-ResourceList-Item__Media" data-test-id="media">
+            {{render-content @media}}
+          </div>
+        {{/if}}
+      </div>
+    {{/if}}
+
+    {{#if (or hasBlock @children)}}
+      <div class="Polaris-ResourceList-Item__Content">
+        {{#if hasBlock}}
+          {{yield}}
+        {{else}}
+          {{render-content @children}}
+        {{/if}}
+      </div>
+    {{/if}}
 
-  {{#if (and shortcutActions (not loading))}}
-    {{#wrapper-element
-      tagName="div"
-      class="Polaris-ResourceList-Item__Actions"
-      click=(action stopPropagation)
-    }}
-      <PolarisButtonGroup
-        data-test-id={{unless persistActions "shortcut-actions"}}
-        @segmented={{not persistActions}}
-        as |buttonGroup|
-      >
-        {{#each shortcutActions as |shortcutAction|}}
-          {{#buttonGroup.item}}
-            {{polaris-button
-              size="slim"
-              plain=persistActions
-              text=shortcutAction.text
-              accessibilityLabel=shortcutAction.accessibilityLabel
-              url=shortcutAction.url
-              external=shortcutAction.external
-              destructive=shortcutAction.destructive
-              icon=shortcutAction.icon
-              loading=shortcutAction.loading
-              disabled=shortcutAction.disabled
-              onClick=(action
-                (or shortcutAction.onAction (action (mut dummy)))
-              )
-            }}
-          {{/buttonGroup.item}}
-        {{/each}}
-      </PolarisButtonGroup>
-    {{/wrapper-element}}
+    {{#if (and @shortcutActions (not this.loading))}}
+      {{#let (element "div") as |ItemActions|}}
```

Comment:
again, for these other `let`'s....can't see a reason why we're doing this....unless I'm wrong and missing something let's just use the elements directly
cc @andrewpye in case you remember why we do this

Reply:
Hm yeah I was just replacing all `wrapper-element`s with the element helper.. but you're right I can't think of why we would have been using `wrapper-element` here in the first place since the `tagName` never changes 🤔Will see if @andrewpye can chime in here but seems like we can drop them now.